internal/controller/config/user_config.go

```diff
@@ -40,6 +41,10 @@ func (s *USDEUserStrategy) UnmarshalJSON(data []byte) (err error) {
 		PPNDStrategyID:        {},
 		NoStrategyID:          {}}
 
+	if strings.TrimSpace(usdeUserStrategyStr) == "" {
+		usdeUserStrategyStr = string(NoStrategyID)
+	}
+
```

Comment:
@juliev0 are you ok with setting NoStrategyID in case the user strategy is unset or if it is empty? I thought this could be acceptable.

Reply:
yes, that's good